gamenews.php

```diff
@@ -6,14 +6,15 @@
     {
         $db = new PDO('sqlite:db/openra.db');
     
-        $insert = $db->prepare("INSERT OR REPLACE INTO sysinfo ('system_id','updated','platform','os','runtime','gl','lang','version','mod','modversion')
-            VALUES (:system_id, :updated, :platform, :os, :runtime, :gl, :lang, :version, :mod, :modversion)"
+        $insert = $db->prepare("INSERT OR REPLACE INTO sysinfo ('system_id','updated','platform','os','x64','runtime','gl','lang','version','mod','modversion')
+            VALUES (:system_id, :updated, :platform, :os, :x64, :runtime, :gl, :lang, :version, :mod, :modversion)"
         );
     
         $insert->bindValue(':system_id', $_REQUEST['id'], PDO::PARAM_STR);
         $insert->bindValue(':updated', date('Y-m-d H:i:s'), PDO::PARAM_STR);
         $insert->bindValue(':platform', $_REQUEST['platform'], PDO::PARAM_STR);
         $insert->bindValue(':os', $_REQUEST['os'], PDO::PARAM_STR);
+        $insert->bindValue(':x64', $_REQUEST['x64'], PDO::PARAM_STR);
```

Comment:
Since sqlite doesn't have a proper bool type, I had to work around that a bit. I hope the line I wrote is proper PHP; I don't know anything about that language.